tests/stat.rs

```diff
@@ -0,0 +1,44 @@
+#![allow(unstable)]
+
+extern crate nix;
+
+#[cfg(test)]
+mod test {
+    use nix::sys::stat::{stat, fstat};
+
+
+    #[test]
+    fn test_stat() {
+
+        use std::old_io::{File, Open, ReadWrite};
+        use std::old_io::fs::{unlink};
+        use std::old_path::posix::Path as OldPath; // temporary until new File type is available that takes std::path::Path
+        use std::path::Path as NewPath;
+
+        let filename = "target/foo.txt";
+        let old_io_path = OldPath::new(filename);  // for creating File
+
+        let test_file = File::create(&old_io_path);
```

Comment:
Maybe you should use the functions present in nix itself to create and unlink files?